source/ci_source/providers/GitHubActions.ts

```diff
@@ -137,11 +137,13 @@ import { readFileSync, existsSync } from "fs"
 export class GitHubActions implements CISource {
   private event: any
 
-  constructor(private readonly env: Env) {
+  constructor(private readonly env: Env, event: any = undefined) {
```

Comment:
I don't see this parameter ever being used?

Reply:
I use it on tests, to avoid to create an event file

Reply:
oh yeah, the var gets set below by the file exists - make sense